textx/metamodel.py

```diff
@@ -243,6 +245,7 @@ def __init__(self, file_name=None, classes=None, builtins=None,
         if file_name:
             file_name = os.path.abspath(file_name)
 
+        # Root path will be dir name of the file if loaded from file.
```

Comment:
This line looks like accidentally copied.

Reply:
line removed